### DISCUSSION TITLE
StatusEffect targets in multiplayer differ from client to server in a specific scenario.

### DISCUSSION THREAD

**Discussion by Sharp-Shark** [FakeFishGames/Barotrauma#13723] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Whilst developing a mod I noticed the behavior of an item from my mod worked one way in singleplayer and another in multiplayer. Basically the actual effects of the status effect triggered if condition A whilst the visual/auditory aspects triggered if condition B. Using lua to assist me, I found out the targets of the statuseffect seem to differ for client to server.

Although I said bug happens in multiplayer hosted through the in-game menu, it also happens in multiplayer hosted by a dedicated server.

Additionally, I have tested and the bug still occurs even after I have uninstalled clientside lua. The bug happens even in vanilla with only the mod I provided installed. The lua is just to get additional debug information.

![barobug1](https://github.com/FakeFishGames/Barotrauma/assets/40040108/ac2b4c13-7c5d-496d-946a-3ceb35cc314e)
[Dam Defense BugReportVersion.zip](https://github.com/FakeFishGames/Barotrauma/files/14863902/Dam.Defense.BugReportVersion.zip)

### Reproduction steps

1. (optional) If you want to get that purple-colored debug info in your debug console just like in the screenshot, install clientside lua.
2. (optional) Add "lua for barotrauma" to your mod list and when you run host server, make sure to pick the correct server executable. Needed to get the lavanda-colored debug info in your debug console.
3. In multiplayer, spawn 2 humans in.
4. Give one of them the enlightened affliction with a high enough strength (I just do "giveaffliction Enlightened 999 [Name]").
5. Spawn in "The 1998".
6. Hit the enlightened NPC and notice the sound and smoke, but item condition doesn't drop to 0 nor does anyone near him, if there is anyone near enough to him, get the enlightened affliction even after repeated triggers.
7. Hit the non-enlightened NPC and notice nothing happens.
8. Give yourself the enlightened affliction with high enough strength.
9. Hit the enlightened NPC and notice the sound, smoke but the item condition **does** drop to 0 and anyone near him, if there is anyone near enough to him, ge the enlightened affliction after repeated triggers.
10. Hit the non-enlightened NPC and notice the item condition drop to 0 aswell as that after repeated hits he will gain the enlightened affliction. After that happens, you'll notice the audio/visual will also happen when hitting him.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/13405. We've got a fix coming up for the issue, but it's unfortunately not yet included in any public build. Sorry for the inconvenience!
